packages/runtime/src/runtime/hooks/useEnsembleAction.tsx

```diff
@@ -89,7 +90,7 @@ export const useExecuteCode: EnsembleActionHook<
       )?.body;
     }
   }, [action, isCodeString, screen]);
-
+  const user = useEnsembleUser();
```

Comment:
const [user] = useEnsembleUser()

You can deconstruct arrays just like useState